src/main/java/graphql/schema/idl/SchemaGenerator.java

```diff
@@ -517,14 +520,17 @@ private TypeResolver getTypeResolverForInterface(BuildContext buildCtx, Interfac
 
 
     private String buildDescription(Node node) {
-        StringBuilder sb = new StringBuilder();
         List<Comment> comments = node.getComments();
+        List<String> lines = new ArrayList<>();
         for (int i = 0; i < comments.size(); i++) {
-            if (i > 0) {
-                sb.append("\n");
+            String commentLine = comments.get(i).getContent().trim();
```

Comment:
I am wary of trimming here.  I think we should use a isBlank() approach.  

For example in markdown they might have used 4 spaces in the description and now we have just trimed them away

How about:

        String commentLine = comments.get(i).getContent()
        if (isBlank(commentLine) {
               ....

and isBlank has the semantics if say : https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-